/usr/bin/diff --ignore-matching-lines=.*$Id:.* -u multimer.c.orig multimer.c
--- multimer.c.orig	Tue Feb 17 19:37:29 1998
+++ multimer.c	Tue Feb 17 19:38:00 1998
@@ -65,6 +65,7 @@ static TQE *timerQ = (TQE *)0;
 /* queue of free Timer Queue Elements */
 static TQE *freeTQEQ = (TQE *)0;
 
+#ifndef timeradd
 void timeradd(struct timeval *a, struct timeval *b, 
   struct timeval *sum)
 {
@@ -77,6 +78,7 @@ void timeradd(struct timeval *a, struct 
     sum->tv_sec = a->tv_sec + b->tv_sec;
   }
 } /* timeradd */
+#endif
 
 
 /*
